recognition/advanced_tree_search.py

```diff
@@ -167,6 +149,7 @@ def __init__(
         lmgc_mem: float = 12.0,
         lmgc_alias: Optional[str] = None,
         lmgc_scorer: Optional[rasr.FeatureScorer] = None,
+        separate_lmi_gc_generation: bool = False,
```

Comment:
I had difficulties parsing `lmi` in my head right away, maybe fully writing `image` is nicer (but not required).